internal/auth0/connection/schema.go

```diff
@@ -881,6 +881,11 @@ var baseConnectionSchema = map[string]*schema.Schema{
 		ForceNew:    true,
 		Description: "Name of the connection.",
 	},
+	"strategy": {
```

Comment:
How was this not picked up by the tests before?

Reply:
Because it was previously defined inline here: https://github.com/auth0/terraform-provider-auth0/pull/648/files#diff-c341ff77267b7ee4a8e7df876275c1809ba3fff0ffc8361e180be1993a456eb8L23

Reply:
Ah 🤦🏻